docs/index.rst

```diff
@@ -37,6 +37,7 @@ Discover also `eXo UX Guidelines <http://exoplatform.github.io/ux-guidelines/>`_
    introduction
    GettingStarted
    PersonalApplications
+   UserReward
```

Comment:
As stated above, I think we should separate wallet, kudos and perk store chapters